build/b88d04fba731603756b1.scss

```diff
@@ -0,0 +1,2 @@
+// extracted by mini-css-extract-plugin
```

Comment:
We should probably add the build folder to `.gitignore`

Reply:
This is still outstanding. Lets first merge #6 and then this PR to get the updated config. Can update the .gitignore at that time

Reply:
added build to gitignore